.circleci/config.yml

```diff
@@ -202,7 +202,7 @@ jobs:
 
   macos:
     macos:
-      xcode: "10.0.0"
+      xcode: "12.0.0-beta"
```

Comment:
Can we change this to the stable release?

Reply:
The beta release needs to keep being used to build macOS arm64 binaries for Big Sur.